hu.json

```diff
@@ -0,0 +1,190 @@
+                "glider" : "Siklórepülő",
+                "emoji" : "Hangulatjel",
+                "loading" : "Töltő képernyő",
+                "outfit" : "Ruházat",
+                "pickaxe" : "Csákány",
+                "skydive" : "Zuhanási nyomvonal",
+                "umbrella" : "Esernyő",
+                "spray" : "Spré",
+                "toy" : "Játék",
+                "pet" : "Háziállat",
+                "music" : "Zene",
+                "wrap" : "Csomagolópapír",
+                "bundle" : "Csomag",
+                "misc" : "Vegyes"
+            },
+            "rarity" : {
+                "legendary" : "Legendás",
+                "epic" : "Epik",
+                "rare" : "Ritka",
+                "uncommon" : "Rendkívüli",
+                "common" : "Közönséges"
+            },
+            "translationNotAvailable" : "Sajnos nincsen elérhető fordítás erre az oldalra.",
+            "translationIncomplete" : "Ezen az oldalon nem az összes szöveg lesz lefordítva."
+        },
+        "nav" : {
+            "home" : "főoldal",
+            "shop" : "bolt",
+            "cosmetics" : {
+                "dropdown" : "kinézetek",
+                "upcoming" : "közelgő",
+                "list" : "lista",
+                "png" : "png",
+                "gallery" : "galléria",
+                "icons" : "ikonok",
+                "reminders" : "emlékeztetők",
+                "history" : "bolt történelem"
+            },
+            "news" : "hírek",
+            "modes" : "módok",
+            "randomiser" : "véletlenszerűsítő",
+            "about" : {
+                "dropdown" : "about",
+                "about" : "fnbr.co-ról",
+                "privacy" : "Adatvédelmi irányelvek",
+                "donate" : "Adományozz",
+                "discord" : "Discord",
+                "twitter" : "Twitter"
+            },
+            "api" : "api",
+            "account" : {
+                "dropdown" : "Fiók",
+                "account" : "Fiók",
+                "logout" : "Kijelentkezés"
+            },
+            "languagePicker" : {
+                "trigger" : "Nyelv",
+                "title" : "Válassz ki egy nyelvet.",
+                "currentLanguage" : "Te most a [Magyar] nyelvet használod."
+            }
+        },
+        "home" : {
+            "title-upcoming" : "Közelgő tételek",
+            "title-all" : "Összes tételek",
+            "error" : "Nem sikerült letölteni az elemeket, kérjük probálkja újra néhány perc múlva.",
+            "error-too-long" : "Ez a vártnál több időt vesz igénybe."
+        },
+        "shop" : {
+            "title" : "Jelenlegi",
+            "title-featured" : "Kiemelt",
+            "title-daily" : "Napi",
+            "error" : "A bolt jelenleg nem elérhető, kérjük probálkozzon néhány perc múlva.",
+            "line1" : "A jelenlegi bolt Fortnite Battle Royale-hoz - Minden nap frissül [00:00 UTC]-kor(https://time.is/UTC|tooltip=Current time in UTC|newtab=true).",
```

Comment:
For the link to be automaticaly inserted the `]` and `(` must be together.
So ` [00:00 UTC]-kor(https://time.is/UTC|tooltip=Current time in UTC|newtab=true)` would become ` [00:00 UTC](https://time.is/UTC|tooltip=Current time in UTC|newtab=true)-kor` I assume?
You can also translate the tooltip text if you wish.